sematic/types/types/matplotlib/figure.py

```diff
@@ -1,21 +1,16 @@
 # Standard Library
-import os
-import uuid
 from typing import Any
 
 # Third-party
 import matplotlib.figure  # type: ignore
+from mpld3 import fig_to_dict
 
 # Sematic
-from sematic.config.config import get_config
 from sematic.types.registry import register_to_json_encodable_summary
 
 
 @register_to_json_encodable_summary(matplotlib.figure.Figure)
 def _mpl_figure_summary(value: matplotlib.figure.Figure, _) -> Any:
-    # TODO: use use artifact binary repr for this instead of by hand
-    filename = uuid.uuid4().hex
-    path = os.path.join(get_config().data_dir, filename)
-    value.savefig(path)
-
-    return {"path": "/data/{}.png".format(filename)}
+    value.set_dpi(70)
```

Comment:
Yet another hack, without setting this to 70 (default 100), a plot will look like this (the font is too small to see):

<img width="766" alt="image" src="https://user-images.githubusercontent.com/1046489/203585180-2eb98ddd-faab-4ee4-af9d-ec9d65242c4e.png">

with DPI 70, it looks like this 

<img width="766" alt="image" src="https://user-images.githubusercontent.com/1046489/203585408-8a3cfd78-ca5c-4df3-a82d-652abd59f09c.png">

It almost looks like a PNG-rendered image. Though they are still essentially different --- The JS way is by laying down SVG elements. The PNG is pixel data. It is almost impossible to make the result of the two approaches identical, if not extremely hard.

Reply:
For reference, this is the PNG-rendered image (the old way)

<img width="763" alt="image" src="https://user-images.githubusercontent.com/1046489/203586868-079678b8-e204-43e0-989b-8d3caf979d5a.png">